packages/plugin-ts-standard-pkg/src/index.ts

```diff
@@ -42,9 +42,14 @@ function getTsConfigPath(options, cwd) {
   return path.resolve(cwd, options.tsconfig || 'tsconfig.json');
 }
 
+function getTscBin(cwd) {
+  return require.resolve('typescript/bin/tsc', {
+    paths: [cwd],
+  });
+}
+
 export async function beforeBuild({cwd, options, reporter}: BuilderOptions) {
-  const tscBin = path.join(cwd, 'node_modules/.bin/tsc');
-  if (!fs.existsSync(tscBin)) {
+  if (!fs.existsSync(getTscBin(cwd))) {
```

Comment:
this will throw an error, instead of returning false/null. Otherwise the PR looks good, so I'll go ahead and make the change so that we can merge